src/support/dump-collection.ts

```diff
@@ -141,98 +141,63 @@ class KuzzleDumper extends JSONLDumper {
   }
 }
 
-export async function dumpCollectionData(sdk: any, index: string, collection: string, batchSize: number, destPath: string, query: any = {}, format = 'jsonl') {
+class CSVDumper extends AbstractDumper {
+  constructor(
+    sdk: any,
+    index: string,
+    collection: string,
+    batchSize: number,
+    destPath: string,
+    query: any = {},
+    protected readonly fields: string[]
+  ) {
+    super(sdk, index, collection, batchSize, destPath, query)
+  }
+
+  protected get fileExtension(): string {
+    return 'csv'
+  }
+  setup(): Promise<void> {
+    return new Promise(resolve => resolve())
+  }
```

Comment:
This could simply be an empty `async` function

Reply:
Ah! Thank you! I was struggling with these signatures! 😅